### PR TITLE
CRAFTERCMS-2643

### PR DESCRIPTION
**Pull request by dejan-brkic**
fixed content type definition and upgrade

### Ticket reference or full description of what's in the PR
https://github.com/craftercms/craftercms/issues/2643

### CODE REVIEW

File: src/main/resources/crafter/studio/upgrade/pipelines.yaml [hunk 1]
```diff
@@ -275,3 +275,12 @@ configurations:
           - type: xsltFileUpgrader
             path: *site-config-tools
             template: crafter/studio/upgrade/update-version.xslt
+      - currentVersion: 2
+        nextVersion: 3
```
Comment (jdrossl) [inline src/main/resources/crafter/studio/upgrade/pipelines.yaml:336]: Please remember to update the version in the file for all blueprints

**Review by jdrossl — COMMENTED** [review]
I think we also need to add upgrade operations to update the `sample-form-control-rte-setup.xml` & `rte-setup.xml` files according to the original PR https://github.com/craftercms/studio/pull/1365/files

File: src/main/resources/crafter/studio/upgrade/pipelines.yaml [hunk 2]
```diff
@@ -209,6 +209,18 @@ pipelines:
         - type: addSiteUuidUpgrader
         - type: versionFileUpgrader
 
+    - currentVersion: 3.1.0.3
+      nextVersion: 3.1.0.4
```
Comment (jdrossl) [inline src/main/resources/crafter/studio/upgrade/pipelines.yaml:238]: Please update `studio_version.xml` in the blueprints to match this version

**Review by jdrossl — CHANGES_REQUESTED** [review]

**Comment by sumerjabri** [comment]
@dejan-brkic bump

**Comment by dejan-brkic** [comment]
fixed PR

**Comment by sumerjabri** [comment]
Closing because it's a bit messy with too many merges.

**Pull request closed without merging**